external-crates/move/crates/move-compiler/tests/move_2024/typing/macros_return_checked_invalid.exp

```diff
@@ -20,7 +20,7 @@ error[E04007]: incompatible types
    │                                          Expected: '&u64'
    ·
 20 │        ref!<u64, u64>(|| &1);
-   │             --- Given: 'u64'
+   │        --------------------- Given: 'u64'
```

Comment:
Any way to get this pointed at the specific type arg again?

Reply:
It's pointing at the body, since that is what ends up getting the u64 at the end. Its a bit of a mess and I'm not really sure I can fix it